sorting/count_sort/count_sort.py

```diff
@@ -0,0 +1,34 @@
+# https://www.ripublication.com/ijaer18/ijaerv13n1_28.pdf
+import math

```

Comment:
Format the comment like this for better readability and remove unused import `math`.

```suggestion
"""
Logic to handle negative numbers in counting sort was referred from the below link:
https://www.ripublication.com/ijaer18/ijaerv13n1_28.pdf
"""
```